erpnext/stock/doctype/warehouse/test_warehouse.py

```diff
@@ -146,4 +181,25 @@ def get_group_stock_account(company, company_abbr=None):
 		if not company_abbr:
 			company_abbr = frappe.get_cached_value("Company", company, 'abbr')
 		group_stock_account = "Current Assets - " + company_abbr
-	return group_stock_account
\ No newline at end of file
+	return group_stock_account
+
+def assign_default_warehouses_to_item(item_code, default_warehouses, company):
+	warehouse = random.choice(default_warehouses)
+	item_default = frappe.new_doc("Item Default")
+	item_default.default_warehouse = warehouse
+	item_default.parent = item_code
+	item_default.company = company
+
+	item_doc = frappe.get_doc('Item', item_code)
+	item_doc.item_defaults = [item_default]
+	item_doc.save()
+
+def create_item(item_code):
```

Comment:
There are *two* similar functions in `test_item.py`, maybe you can just reuse them.